sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -71,15 +85,15 @@ class SnackBar extends Component {
     ];
     if (actions != null)
       children.addAll(actions);
-
     return new SlideTransition(
```

Comment:
I don't understand how this would work, then. How would the transition get the performance? You're only supposed to pass data to a Widget in its constructor.